build.gradle

```diff
@@ -14,6 +14,7 @@ String cldrdir = buildDir.toString() + '/cldr-data/' + cldrVersion.toString()
 System.setProperty("CLDR_DIR", cldrdir.toString())
 
 repositories {
+    mavenLocal()
```

Comment:
remove this for merge, we want to use upstream stuff i think

Reply:
Yes sure